Failing test: x-pack/solutions/security/test/security_solution_cypress/cypress/e2e/investigations/alerts/event_rendered_view.cy.ts

https://buildkite.com/elastic/kibana-on-merge/builds/77983#019985ef-5b8f-40bd-bf5e-032a4ebaf3c7
```
  1) Event Rendered View
       should show Event Summary column correctly:
     AssertionError: Timed out retrying after 150000ms: Expected to find element: `[data-gridcell-column-id="eventSummary"]`, but never found it.
      at Context.eval (webpack:///./e2e/investigations/alerts/event_rendered_view.cy.ts:49:33)
  2) Event Rendered View
       should show TopN in Event Summary column:
     AssertionError: Timed out retrying after 150000ms: Expected to find element: `[data-test-subj="alertFieldBadge"] [data-test-subj="render-content-host.name"]`, but never found it.
      at showHoverActionsEventRenderedView (webpack:///./tasks/alerts.ts:584:5)
      at Context.eval (webpack:///./e2e/investigations/alerts/event_rendered_view.cy.ts:54:38)
```

--- a/x-pack/solutions/security/test/security_solution_cypress/cypress/e2e/investigations/alerts/event_rendered_view.cy.ts
+++ b/x-pack/solutions/security/test/security_solution_cypress/cypress/e2e/investigations/alerts/event_rendered_view.cy.ts
@@ -26,6 +26,7 @@ import {
   waitForAlerts,
 } from '../../../tasks/alerts';
 import { createRule } from '../../../tasks/api_calls/rules';
+import { deleteAlertsAndRules } from '../../../tasks/api_calls/common';
 import { login } from '../../../tasks/login';
 import { visit } from '../../../tasks/navigation';
 import { ALERTS_URL } from '../../../urls/navigation';
@@ -37,15 +38,18 @@ import {
 
 describe(`Event Rendered View`, { tags: ['@ess', '@serverless'] }, () => {
   beforeEach(() => {
+    deleteAlertsAndRules();
     login();
     createRule(getNewRule());
-    visit(ALERTS_URL);
-    waitForAlerts();
-    // Set up intercept before switching views to avoid a race condition where the
-    // search request starts after waitForNetworkIdle's 500ms idle window has passed.
+    // Set up intercept before visiting the page so it captures all search
+    // requests, including those triggered by switchAlertTableToEventRenderedView.
+    // This avoids a race condition where the request completes before the alias
+    // is registered, causing cy.wait('@alertsSearch') to time out.
     cy.intercept('POST', '/internal/search/privateRuleRegistryAlertsSearchStrategy').as(
       'alertsSearch'
     );
+    visit(ALERTS_URL);
+    waitForAlerts();
     switchAlertTableToEventRenderedView();
     cy.wait('@alertsSearch');
     waitForAlerts();

